src/components/Slider.js

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import {
+  StyleSheet,
+  ScrollView,
+  Text,
+  View,
+  Image,
+  TouchableOpacity
+} from 'react-native'
+import colors from '../theme.json'
+import globalStyles from '../globalStyles'
+import Icon from 'react-native-vector-icons/MaterialIcons'
+import Icon2 from 'react-native-vector-icons/MaterialCommunityIcons'
+
+const slideColors = {
+  RED: 'red',
+  YELLOW: 'gold',
+  GREEN: 'green'
+}
+
+class Slider extends Component {
+  state = {
+    selectedColor: colors.green
+  }
+  render() {
+    return (
+      <View>
+        <ScrollView
+          horizontal
+          showsHorizontalScrollIndicator={false}
+          contentContainerStyle={{
+            width: 850,
+            flexGrow: 1,
+            flexDirection: 'row',
+            justifyContent: 'space-between'
+          }}
+          ref="_scrollView"
```

Comment:
String refs are considered problematic and deprecated: https://reactjs.org/docs/refs-and-the-dom.html#legacy-api-string-refs. Consider using this:

```suggestion
          ref={comp => { this._scrollView = comp }}
```
And then call `this._scrollView.scrollTo` bellow instead of `this.refs._scrollView.scrollTo`. You could instead use the documentation approach, but then you also need to add `this._scrollView = React.createRef()` to the constructor. This way is the quickest.